content/docs/lifting-state-up.md

```diff
@@ -299,32 +299,31 @@ class Calculator extends React.Component {
-* Inside these methods, the `Calculator` component asks React to re-render itself by calling `this.setState()` with the new input value and the current scale of the input we just edited.
-* React calls the `Calculator` component's `render` method to learn what the UI should look like. The values of both inputs are recomputed based on the current temperature and the active scale. The temperature conversion is performed here.
-* React calls the `render` methods of the individual `TemperatureInput` components with their new props specified by the `Calculator`. It learns what their UI should look like.
-* React calls the `render` method of the `BoilingVerdict` component, passing the temperature in Celsius as its props.
-* React DOM updates the DOM with the boiling verdict and to match the desired input values. The input we just edited receives its current value, and the other input is updated to the temperature after conversion.
+* React-ը կանչում է DOM `<input>`-ի վրա որպես `onChange` սահմանված ֆունկցիան։ Մեր պարագայում դա `TemperatureInput` կոմպոնենտի `handleChange` մեթոդն է։
+* `TemperatureInput` կոմպոնենտի `handleChange` մեթոդը կանչում է `this.props.onTemperatureChange()`-ը նոր արժեքով։ Նրա prop-երը, ներառված `onTemperatureChange`, տրամադրված էին իր ծնող\` `Calculator` կոմպոնենտի կողմից։
+* Երբ այն նախորդ անգամ արտապատկերվեց, `Calculator`-ը նշանակեց Ցելսուսի `TemperatureInput`-ի `onTemperatureChange`-ը `Calculator`-ի `handleCelsiusChange` մեթոդով և համապատասխանաբար, Ֆարենհեյթի `TemperatureInput`-ի `onTemperatureChange`-ը `Calculator`-ի `handleFahrenheitChange` մեթոդով։ `Calculator`-ի այս երկու մեթոդներից ցանկացածը կկանչվի կախված, թե որ input-ն է խմբագրվել։
+* Այս մեթոդների ներսում, `Calculator` կոմպոնենտը խնդրում է React-ին վերա-արտակատկերել ինքն իրեն\` կանչելով `this.setState()` հենց նոր խմբագրած input-ի ընթացիկ սանդղակով և input-ի արժեքով։
+* React-ը կանչում է `Calculator` կոմպոնենտի `render` մեթոդը, որպեսզի իմանա թե ինչ տեսք պետք է ունենա UI-ը։ Երկու input-ների արժեքները վերա-հաշվարկված են\` հիմնված ընթացիկ ջերմաստիջանի և ակտիվ սանդղակի վրա։ Այստեղ է կատարվում ջերմաստիջանի փոխակերպումը։
+* React-ը կանչում է ամեն մի ինդիվիդուլ `TemperatureInput` կոմպոնենտի `render` մեթոդը նրանց նոր prop-երով\` սահմանված `Calculator`-ի կողմից։ Այն հասկանում է, թե ինչ տեսք պետք է ունենա նրանց UI-ը։
+* React-ը կանչում է `BoilingVerdict` կոմպոնենտի `render` մեթոդը\` փոխանցելով ջերմաստիճանը ցելսուսով որպես նրա prop։
+* ReactDOM-ը թարմացնում է DOM-ը «եռման որոշմամբ», որը հիմնված է input-ների արժեքների վրա։ input-ը, որը հենց նոր խմբագրեցինք, ստանում է իր ընթացիկ արժեքը, իսկ մյուսը թարմանում է ջերմաստիճանի փոխակերպումից հետո։
```

Comment:
* React DOM updates the DOM with the boiling verdict and to match the desired input values. The input we just edited receives its current value, and the other input is updated to the temperature after conversion.